eng/docker/libraries-sdk.linux.Dockerfile

```diff
@@ -8,7 +8,7 @@ WORKDIR /repo
 COPY . .
 
 ARG CONFIGURATION=Release
-RUN ./build.sh -ci -subset clr+libs -runtimeconfiguration release -c $CONFIGURATION
+RUN ./build.sh clr.buildtools+clr.runtime+clr.corelib+clr.nativecorelib+libs -ci -rc release -c $CONFIGURATION
```

Comment:
why are you changing this?

Reply:
The build were failing when building nuget packages in coreclr, I wanted to iterate quicker as CI was slow to test my changes. We can revert back... anyway, you know my take on building not necessary stuff on this builds.

Reply:
FWIW,  https://github.com/dotnet/runtime/commit/11bf4b8f6e96a1b9deaef6470201d4e10f720b02#diff-f1921dfe4ae191cbfc80c608aa17dd4a would have broken this pipeline if we have had encoded the individual subsets here instead of just using the parent one `clr`. I'm fairly certain it wouldn't have been caught in the PR and we would have fixed it reactively. Just something that underlines my take on this.

Reply:
Yeah that was a breaking change that no one was aware of... it should've been done through the batched rollouts and now that subset is being removed... I understand the point, I'm just trying to help fix these builds without investing much time. Once the build is green, we can revert this change and investigate why building the coreclr packages fail on  these builds.